### ISSUE TITLE
What does this mod do?

### ISSUE THREAD

**Issue by Craluminum2413** [Rahjital/VSBullseye#1]

**Comment by TeacupAngel** [comment]
The main goals of the mod were to make the balance of ranged weapons more consistent, and to make archery more skill-based than it is by default.

In vanilla VS, arrows had the issue that at higher metal tiers, they were becoming extremely inefficient compared to spears, [as shown in this table](https://docs.google.com/spreadsheets/d/1N4Td2SsHkUqhneZpuasqhhpkrlic7UBhlDXvPwRaDUo/edit?usp=sharing). The mod fixed the efficiency of spears at 200% of arrows no matter the tier, to make the cost advantage of spears more reasonable.

The spear throw attack was also easily spammable and started with high damage at flint but scaled very poorly with better materials, which in some ways made the flint spear the best one even for well equipped characters. I made high-tier spears deal more damage, and added cooldown to ranged attacks to stop spear spam from dominating fights so much.

In vanilla the bow is also fairly inaccurate, and past about 50 blocks it can be entirely up to RNG whether you hit or miss. The mod replaced this inaccuracy with mouse drift, so that a good player could compensate and shoot accurately even at long distances. After a while of getting used to it, I could hit a bighorn sheep at 80 blocks fairly reliably, and I'm sure a good archer could stretch it a little further still. I also made the spear a little more short-ranged, but mostly removed its accuracy penalty when running to make up for it.

Unfortunately I didn't finish the mod because 1.14 dropped as I was working on it, and I didn't have the time to test how the mod would be affected by the addition of steel and classes. I also wasn't quite happy with how the mouse drift and cooldown worked, those could use some improvement as well.
